### PR TITLE
Postgis chart

### PR DESCRIPTION
**Pull request by roymx**
Added postgis chart
meectl can deploy/delete as dependency

### CODE REVIEW

File: charts/postgis/values.yaml [hunk 1]
```diff
@@ -0,0 +1,565 @@
...
+## PostgreSQL Master parameters
+##
+master:
+  ## Node, affinity, tolerations, and priorityclass settings for pod assignment
+  ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector
+  ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+  ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#taints-and-tolerations-beta-feature
+  ## ref: https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption
+  nodeSelector: {}
+  affinity: {}
```
Comment (dilallkx) [inline charts/postgis/values.yaml:341]: To support hostpath PV in cluster setup, add affinity here:
```
affinity:
  nodeAffinity:
    requiredDuringSchedulingIgnoredDuringExecution:
      nodeSelectorTerms:
      - matchExpressions:
        - key: node-role.kubernetes.io/master
          operator: Exists
```

Comment (dilallkx) [reply charts/postgis/values.yaml:341]: I missed this in previous OMT review; add affinity in OMT chart as well.

File: charts/postgis/values.yaml [hunk 2]
```diff
@@ -0,0 +1,565 @@
...
+## PostgreSQL Slave parameters
+##
+slave:
+  ## Node, affinity, tolerations, and priorityclass settings for pod assignment
+  ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector
+  ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+  ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#taints-and-tolerations-beta-feature
+  ## ref: https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption
+  nodeSelector: {}
+  affinity: {}
```
Comment (dilallkx) [inline charts/postgis/values.yaml:392]: Slaves also depend on hostpath PV. Add master node affinity here as well.

File: go-apps/meepctl/cmd/deploy.go [hunk 3]
```diff
@@ -200,6 +202,8 @@ func deployRunScriptsAndGetFlags(targetName string, chart string, cobraCmd *cobr
 		flags = utils.HelmFlags(nil, "--set", "persistentVolume.location="+deployData.workdir+"/couchdb/")
 	case "meep-open-map-tiles":
 		flags = utils.HelmFlags(nil, "--set", "persistentVolume.location="+deployData.workdir+"/omt/")
+        case "meep-postgis":
```
Comment (dilallkx) [inline go-apps/meepctl/cmd/deploy.go:205]: Fix indentation.

File: charts/postgis/values.yaml (hunk 1 above)
Comment (roymx) [reply charts/postgis/values.yaml:341]: - Postgis: done for master and slave
- OMT: done

File: charts/postgis/values.yaml (hunk 2 above)
Comment (roymx) [reply charts/postgis/values.yaml:392]: done

File: go-apps/meepctl/cmd/deploy.go (hunk 3 above)
Comment (roymx) [reply go-apps/meepctl/cmd/deploy.go]: done

**Pull request merged**